crates/bevy_color/src/color.rs

```diff
@@ -17,7 +18,7 @@ use derive_more::derive::From;
 ///
 /// # Operations
 ///
-/// [`Color`] supports all the standard color operations, such as [mixing](Mix),
+/// [`Color`] supports all the standard color operations, such as [interpolating](bevy_math::Interpolate),
```

Comment:
I do find it a little funny that colour mixing is now a bevy_math thing.
I wonder what bevy_color people think about this change.
Also I think it would be nice to add in an alias here for discoverability.
```suggestion
/// [`Color`] supports all the standard color operations, such as [mixing/interpolating](bevy_math::Interpolate),
```

Reply:
I would be fine to add a `mix` and `mix_assign` alias to `Interpolate` to ease migrations.